api/proto/teleport/legacy/types/types.proto

```diff
@@ -8265,3 +8269,27 @@ message AccessGraphAzureSync {
   // Integration is the integration name used to generate credentials to interact with AWS APIs.
   string Integration = 2 [(gogoproto.jsontag) = "integration,omitempty"];
 }
+
+// TargetHealth describes the health status of network connectivity between
+// an agent and a resource.
+message TargetHealth {
```

Comment:
are we adding timestamp of the last check after all the discussions on performance?

Reply:
yes. i think "last checked" is quite valuable even if we cannot refresh it often. but since it's left out from the RFD, it's up to you.